frontend/src/components/ProjectStudentComponent.tsx

```diff
@@ -1,36 +1,221 @@
 import {useTranslation} from 'react-i18next';
-import {make_submission} from "../utils/api/Submission.ts";
+import {RegularButton} from "./RegularButton.tsx";
+import {IoExitOutline} from "react-icons/io5";
+import {MdOutlinePersonAddAlt1} from "react-icons/md";
 import {Submission} from "../utils/ApiInterfaces.ts";
 import {DEBUG} from "../pages/root.tsx";
-import {RegularButton} from "./RegularButton.tsx";
+import {make_submission} from "../utils/api/Submission.ts";
+import {joinGroup, leaveGroup} from "../utils/api/Groups.ts";
+import {getGroupInfo, loadGroupMembers} from "../dataloaders/ProjectsStudentLoader.ts";
 
-export default function ProjectStudentComponent(props: { project: ProjectStudent }): JSX.Element {
+function ProjectInfo(props: { project: ProjectStudent }): JSX.Element {
+    const {t} = useTranslation();
 
-    // true als er een groep is, anders false.
-    const is_in_group = props.project.groupMembers && props.project.groupMembers.length > 0;
+    return (
+        <div className={"pt-5"}>
+            <FieldWithLabel fieldLabel={t('project.name')} fieldBody={props.project.projectName} arrow={true}/>
+            <FieldWithLabel fieldLabel={t('project.course')} fieldBody={props.project.courseName} arrow={true}/>
+            <FieldWithLabel fieldLabel={t('project.deadline')} fieldBody={props.project.deadline} arrow={true}/>
+            <div className="field is-horizontal">
+                <div className="field-label">
+                    <label className="label">{"> "}Status: </label>
+                </div>
+                <div className="field-body field">
+                    {props.project.status == ProjectStatus.FAILED &&
+                        <label className={"has-text-danger"}>{t('project.failed')}</label>}
+                    {props.project.status == ProjectStatus.SUCCESS &&
+                        <label className={"has-text-success"}>{t('project.success')}</label>}
+                    {props.project.status == ProjectStatus.PENDING &&
+                        <label>—</label>}
+                </div>
+            </div>
+            <FieldWithLabel fieldLabel={"> " + t('project.description')}
+                            fieldBody={props.project.description} arrow={false}/>
+            <div className="field is-horizontal">
+                <div className="field-label">
+                    <label className="label">{"> " + t('project.submission_files') + ":"}</label>
+                </div>
+                <div className="field-body">
+                    <div className="field">
+                        { // TODO requiredFiles moet een string geven zoals bij teacher voor dit kan werken
+                            /*<SimpleTests
+                                teacherOrStudent={TeacherOrStudent.STUDENT}
+                                initialData={}
+                                setData={undefined}
+                                setHasChanged={undefined}
+                            /> */}
```

Comment:
Moet dit er nog instaan? anders kan je dit mss verwijderen?

Reply:
Ja, da geeft nu een error maar zou moeten opgelost worden met #320

Reply:
(staat nu ook gecomment in main)